analytics_data_api/v0/serializers.py

```diff
@@ -332,6 +332,14 @@ class LearnerSerializer(serializers.Serializer, DefaultIfNoneMixin):
     enrollment_date = serializers.DateField(source='enrollment_date', format=settings.DATE_FORMAT)
     cohort = serializers.CharField(source='cohort')
 
+    def transform_segments(self, _obj, value):
+        # returns null instead of empty strings
+        return value or None
+
+    def transform_cohort(self, _obj, value):
+        # returns null instead of empty strings
+        return value or None
```

Comment:
It turns out that the field defaults to `''` when `null`.  We could create extend the elasticsearch dsl's `String` field type.  However, I'm not sure that I see the use case for an empty string cohort.  Does that happen?